DEF-2971 - Updated emscripten version to 1.38.12.

andsveking opened the pull request.

andsveking committed:
DEF-2971 - Updated emscripten version to 1.38.12.

andsveking committed:
Removed old workarounds for emscripten SDK on Linux.

andsveking committed:
Updated build.yml to work with emscripten updates to extender.

mathiaswking reviewed (approved):
Looks good, but I'd like some more investigation to keep the separation between Dockerfile/Extender/build.yml a bit cleaner.

andsveking committed:
Updated build.yml on review feedback.

Emscripten binary path is no longer hardcoded, it will be picked up from
environment variable instead.
